docs/safe-haven-services/s3-service.md

```diff
@@ -0,0 +1,145 @@
+# S3 Service
+
+There is no general-purpose S3 service within Safe Haven Services, unlike the [EIDF S3 service](../services/s3/).
+
+However there is a S3 service in SHS with the following caveats:
+* it is only available to the Scottish National Safe Haven, other tenants by arrangement
+* it is a read-only service, as a way of providing access to large collections of files
+* it is not a storage solution for users wanting to create their own files
+
+## Access arrangements
+
+Access to buckets is via keys (Access Key and Secret Access Key) provided to the user by the Research Coordinator.
+
+## How to use the service
+
+To access files you need the following information:
+
+* Region is "us-east-1"
+* Endpoint URL is "http://nsh-fs02:7070"
+* Access key
+* Secret access key
+* The web proxy variables must be empty
+
+For example when using a command-line or script:
+
+|Environment variable   |Value                 |
+|-----------------------|----------------------|
+| AWS_DEFAULT_REGION    | us-east-1            |
+| AWS_ENDPOINT_URL      | http://nsh-fs02:7070 |
+| AWS_ACCESS_KEY_ID     | as provided by RC    |
+| AWS_SECRET_ACCESS_KEY | as provided by RC    |
+| http_proxy            | no value             |
+
+```
+export AWS_DEFAULT_REGION=us-east-1
+export AWS_ENDPOINT_URL=http://nsh-fs02:7070
+export AWS_ACCESS_KEY_ID=put_your_key_here
+export AWS_SECRET_ACCESS_KEY=put_your_secret_here
+export http_proxy=
+```
```

Comment:
Is there a better, more permanent solution within the constraints of the user? For example, is it possible to run `aws configure --profile minio` and add the endpoint to ~/.aws/config?

Reply:
To discuss for future improvement. The docs can go ahead as they are just now. Can we say between the table and the code "Create an env file with the following variables:" and after "Apply the configuration by running `source s3.env`" or similar?